lib/GuestManager.php

```diff
@@ -270,8 +286,21 @@ public function sendEmailInvitation(Room $room, Participant $participant): void
 			);
 		}
 
-		$template->addFooter();
+		if ($room->getLobbyState() !== Webinary::LOBBY_NONE && $room->getLobbyTimer() !== null) {
+			$template->addBodyText($this->l->t('Click the button below to join the lobby now.'));
```

Comment:
you need to provide the second argument (plain text) and provide a different text for plain text emails as there is no button there.
Also we should add a line break somehow (see screenshot on other comment)